docs/fabric-renderer.md

```diff
@@ -3,26 +3,26 @@ id: fabric-renderer
 title: Fabric Rendering
 ---
 
-Fabric is React Native's new rendering system. It is designed to improve interoperability with mobile platforms and to unlock new capabilities. Fabric is a conceptual evolution of the render system currently used by React Native. The core principle is to unify more render logic in C++ versus spread across the different [host platforms](glossary#host-platform).
+Fabric is React Native's new rendering system. It was designed, built, and rolled out at Meta between 2018 and 2021. It is conceptual evolution of the legacy render system. The core principles are to unify more render logic in C++, improve interoperability with [host platforms](glossary#host-platform), and to unlock new capabilities.
 
-This documentation provides an overview of the [Fabric renderer](glossary#fabric-render) and its concepts. It avoids platform specifics and doesn’t contain any code snippets or pointers. This documentation covers key concepts, motivation and benefits, and an overview of the render pipeline in Fabric in different scenarios.
+This documentation provides an overview of the [new renderer](glossary#fabric-render) and its concepts. It avoids platform specifics and doesn’t contain any code snippets or pointers. This documentation covers key concepts, motivation, benefits, and an overview of the render pipeline in different scenarios.
 
-## Motivations and Benefits of Fabric
+## Motivations and Benefits of the new renderer
 
-The Fabric render architecture was created to unlock better user experiences that weren’t possible with the legacy architecture. Some examples include:
+The render architecture was created to unlock better user experiences that weren’t possible with the legacy architecture. Some examples include:
 
-- With improved interoperablity between [host views](glossary#host-view-tree-and-host-view) and React views, Fabric is able to to measure and render React surfaces synchronously. In the legacy architecture, React Native layout was asynchronous which led to a layout “jump” issue when embedding a React Native rendered view in a host platform view.
-- With Fabric’s support of multi-priority and synchronous events, the renderer can prioritize certain user interactions to ensure they are handled in a timely manner.
+- With improved interoperablity between [host views](glossary#host-view-tree-and-host-view) and React views, the renderer is able to to measure and render React surfaces synchronously. In the legacy architecture, React Native layout was asynchronous which led to a layout “jump” issue when embedding a React Native rendered view in a _host view_.
```

Comment:
Question about italics, is the idea to italicize every word/concept that's in the glossary that isn't linked to the definition?